intuita-webview/src/utilities.ts

```diff
@@ -0,0 +1,2 @@
+export const generateRandomColor = () =>
```

Comment:
Colors should not be random, because this provides no reproducibility.

One idea is to create a fixed palette (e.g. 32 colors), create a hash of whatever object you want to assign a color to, convert the hash to a number (perhaps the least significant byte of the hash?) and then modulo divide it by 32, getting a color.

Reply:
addressed